cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1414,6 +1448,81 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     );
                 });
 
+                describe('Show correctness is set as expected from subsection metadata.', function() {
+                    var setUp = function(metadata) {
+                        var mockCourseJSON = createMockCourseJSON({}, [
+                            createMockSectionJSON({}, [
+                                createMockSubsectionJSON(metadata, [])
+                            ]),
+                        ]);
+                        createCourseOutlinePage(this, mockCourseJSON, false);
+                        outlinePage.$('.outline-subsection .configure-button').click();
+                        selectAdvancedSettings();
+                    };
+                    it('Show correctness "always" (default)', function() {
+                        setUp({
+                            show_correctness: 'always',
+                        });
+                        expectShowCorrectness('always');
```

Comment:
@pomegranited you can do something like

    var correctness_values = ['always', 'never', 'past_due', ...];
    for (var i=0; i < correctness_values; i++) {
        var correctness_value = correctness_values[i];
        it("show correctness (" + correctness_value + ")", function() { ... })
    }